enterprise-suite/gotests/tests/portforward/portforward_test.go

```diff
@@ -0,0 +1,71 @@
+package ingress
+
+import (
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"testing"
+
+	"github.com/lightbend/gotests/args"
+	"github.com/lightbend/gotests/testenv"
+
+	"github.com/lightbend/gotests/util"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+const consoleRemotePort = 8080
+const consoleDeployment = "deployment/es-console"
+
+// Hardcode the local port because with random port setup it's currently
+// tricky to get cmd.StartAsync stdout without stopping it first.
+const localPort = 58363
```

Comment:
Can you copy the logic at https://github.com/sky-uk/merlin/blob/master/e2e/e2e.go#L257?

We can stick it in util somewhere. It provides the ability to discover an open port on the local machine.

Reply:
Ok, added utility function `FindFreePort`